src/features/feeds/components/FeedList.tsx

```diff
@@ -21,7 +21,7 @@ export const FeedList = ({
   ecosystem?: string
   initialCache?: Record<string, ChainMetadata>
 }) => {
-  const chains = ecosystem === "deprecating" ? ALL_CHAINS : ecosystem === "solana" ? SOLANA_CHAINS : CHAINS
+  const chains = CHAINS
```

Comment:
why dont we just use the `CHAINS` object?

Reply:
Yes this should work now that chains.ts is simplified. I'll change it.

Reply:
Looks like we still need this for the feed deprecation lists. I restored part of it.